[blog] Danilo Leal joins Material-UI (#27231)
mui-org/material-ui: docs/pages/blog/danilo-leal-joining.md

---
title: Danilo Leal joins Material-UI
description: We are excited to share that Danilo Leal has joined Material-UI.

> date: 2021-07-14T00:00:00.000Z

Author: ```suggestion
date: 2021-07-15T00:00:00.000Z
```

authors: ['oliviertassinari']
card: true
---

We are excited to share that [Danilo Leal](https://daniloleal.co/) has joined Material-UI!
He started a couple of days ago as a lead designer.

Before joining Material-UI, he worked as a designer at Loggi, a Brazilian logistics unicorn.

> He has spent the last three years diving into product design, product strategy, and design systems.

Reviewer: ```suggestion
He has spent the last five years diving into product design, product strategy, and design systems.
```

> Danilo strongly believes that designers should learn how to code, get their hands dirty.

Reviewer: ```suggestion
Danilo strongly believes that code is a fundamental piece of a designer's work. It is, after all, the real medium through where users experience the product.
```
Expanding a bit on this :)

> For instance, he drove deep into the codebase of Material-UI to fulfill the customization needs of Loggi.

Reviewer: ```suggestion
For instance, one of his continuous contributions to Loggi was deep-diving into Material-UI's codebase to fulfill Loggi's customization needs.
```

His aspirations are well aligned with where we want to push the company in the long term.
He wants to challenge the status quo.
He wants to push the envelope around how products are built.
We believe there are two high potential opportunities to explore: 1. how designers and developers collaborate, 2. how accessible is creating custom UIs.

> His design [challenge assignment](https://www.figma.com/file/UPgXH2mdWqV8QzAuN5VM37/Danilo-assignment) (done under a few hours) gave up confidence that Danilo is exactly who the role needs.

Reviewer 2: ```suggestion
His design [challenge assignment](https://www.figma.com/file/UPgXH2mdWqV8QzAuN5VM37/Danilo-assignment) (done under a few hours) gave us confidence that Danilo is exactly who the role needs.
``` 
?

Reviewer: Oh wow haha. On point correction 😅
"Gave up" changes the meaning completely haha!

Reviewer 2: I hope it’s ‘gave us’ 😁
